tensorflow/python/data/experimental/kernel_tests/data_service_test_base.py

```diff
@@ -115,7 +115,16 @@ def stop_dispatcher(self):
 
   # pylint: disable=protected-access
   def restart_dispatcher(self):
-    """Stops `dispatcher` and creates a new dispatcher with the same port."""
+    """Stops `dispatcher` and creates a new dispatcher with the same port.
+
+    When the dispatcher is restarted with `fault-tolerant_mode=False`,
```

Comment:
This scenario is too specific, can we make the docstring more general, e.g. "Restarting is only supported when the dispatcher is configured with `fault_tolerant_mode=True`"?